panacea/web/static/js/view.js

```diff
@@ -139,6 +156,45 @@ export const restoreFileForm = () => {
   hideElement(loadingElement);
 };
 
+export const generateErrorHighlight = (code, errorLine, numContextLines = 2) => {
+  const lines = code.split('\n');
```

Comment:
Do you think it makes sense to split the lines once we get the response? To avoid splitting multiple times.